Dokan/Dokan/Scenes/Registration/RegisterViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RegisterViewModel.swift
+//  Dokan
+//
+//  Created by heba isaa on 05/07/2022.
+//
+
+import Foundation
+
+class RegisterViewModel {
+        private var emailOrPhoneTxt: String = ""
```

Comment:
We should used full names not shortcuts. This is best practice for Swift. 
```suggestion
        private var emailOrPhoneText: String = ""
```